src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  align-items: center;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+
+  &__star {
+    display: flex;
+
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    background-size: initial;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  @for $star from 1 through 4 {
+    &__star--#{$star} {
+      background-image: url(../images/star-active.svg);
```

Comment:
Why it's 4? And actually, it doesn't work, it should be the same functionality as in the stars task. if the block has `stars--3z modifier you should show 3 active stars and so on